src/public/js/vino_US.js

```diff
@@ -1657,6 +1661,38 @@ $(window).on("popstate", function () {
     tvii.utils.changePage(location.search, true);
 });
 
+tvii.router.connect("^[?&]page=manual(?:&|$)", function () {
+    function changeScreen(hide, show, id) {
+        $(hide).addClass("none");
+        $(show).removeClass("none");
+        $(document).trigger("modalchange:manual", [$(show)]);
+        
+        history.pushState(null, "", location.search + "&manual=" + id);
+    };
+
+    $("[data-show]").on("click", function () {
+        const hide = $(this).attr("data-hide");
+        const show = $(this).attr("data-show");
+        const id = $(this).attr("data-id");
+        
+        changeScreen(hide, show, id);
+    });
+
+    $(".manual-back").on("click", function () {
+        vino.soundPlay("SE_A_CLOSE_TOUCH_OFF");
+        history.back();
+    });
+
+    $(function () {

```

Comment:
maybe add comments or something, I don't understand what this does - unless it just adds the manualBody contents to the body?